db/migrate/20160718020648_create_focus_areas.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFocusAreas < ActiveRecord::Migration
+  def change
+    create_table :focus_areas do |t|
```

Comment:
Hey @erinnachen what exactly are the focus areas? Are they the work stations within the groves?